src/fragments/start/getting-started/android/setup.mdx

```diff
@@ -113,10 +113,10 @@ dependencies {
 
     Android Studio requires you to sync your project with your new configuration. To do this, click **Sync Now** in the notification bar above the file editor.
 
-    ![](/images/lib/getting-started/android/set-up-android-studio-sync-gradle.png)
+    ![Shows where to click for syncing with gradle in Android studio ](/images/lib/getting-started/android/set-up-android-studio-sync-gradle.png)
```

Comment:
Similar to the gradle comments above, we could have this image just be decorative (since the text above mentions how to find the sync now button) or show with words what is going on in the picture